CRM/Contribute/BAO/Contribution.php

```diff
@@ -5669,4 +5674,79 @@ public static function replaceContributionTokens(
     return $contributionDetails;
   }
 
+  /**
+   * Record adjusted amount.
+   *
+   * @param int $updatedAmount
+   * @param int $paidAmount
+   * @param int $contributionId
+   *
+   * @param int $taxAmount
+   * @param bool $updateAmountLevel
+   *
+   * @return bool|\CRM_Core_BAO_FinancialTrxn
+   */
+  public static function recordAdjustedAmt($updatedAmount, $paidAmount, $contributionId, $taxAmount = NULL, $updateAmountLevel = NULL) {
```

Comment:
Actually, I see this is just a helper function to split out some code below. Could we make it a protected private class and name it beginning with an underscore https://www.drupal.org/docs/develop/coding-standards/object-oriented-code#naming point 9 ?